File: Programming_Task_03_ShaliniSengar/Python_Files/Number_Analysis_Tool.py
n = int(input("Enter a number: "))

sum_numbers = 0
even_count = 0
odd_count = 0

for i in range(1, n + 1):
    sum_numbers += i

    if i % 2 == 0:
        even_count += 1
    else:
        odd_count += 1

print("Sum of numbers from 1 to", n, "=", sum_numbers)
print("Count of even numbers =", even_count)
print("Count of odd numbers =", odd_count)

File: Programming_Task_03_ShaliniSengar/C Files/Number_Analysis_Tool.c
#include <stdio.h>

int main() {
    int N, i;
    int sum = 0, evenCount = 0, oddCount = 0;

    printf("Enter a number: ");
    scanf("%d", &N);

    for(i = 1; i <= N; i++) {
        sum += i;

        if(i % 2 == 0)
            evenCount++;
        else
            oddCount++;
    }

    printf("Sum = %d\n", sum);
    printf("Even Numbers = %d\n", evenCount);
    printf("Odd Numbers = %d\n", oddCount);

    return 0;
}